extension/js/newResponseNotification.js

```diff
@@ -1,63 +1,85 @@
-(function(){
-    let currentCount = document.getElementsByClassName("post has-profile").length;
-    let lastPost = document.getElementsByClassName("post has-profile")[currentCount -1]
+(function () {
+  let currentCount = document
+    .querySelector("div.pagination")
+    .innerHTML.match(/(\d.) תגובות/)[1];
```

Comment:
I think you mean this  `\d.` only selects 2 digit number... failed on https://www.ivelt.com/forum/viewtopic.php?p=4855050#p4855050
```suggestion
    .innerHTML.match(/(\d+) תגובות/)[1];
```

Reply:
Typo (\d. also matches non digit numbers. \d+ matches any amount of digits)